app/src/pages/organization/organizationProjectsPage/modals/renameProjectModal/renameProjectModal.tsx

```
@@ -66,7 +66,8 @@ const RenameProjectModal: FC<RenameProjectModalProps> = ({
 
   const okButton: ModalButtonProps = {
     children: formatMessage(COMMON_LOCALE_KEYS.RENAME),
-    onClick: handleSubmit(onSubmit),
+    // @Alla_Prischepa check it please
+    onClick: () => handleSubmit(onSubmit) as void,
```

Biome (1.9.4)

[error] void is confusing outside a return type or a type parameter.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)